doc/DataPlaneCodeGeneration/Handle_Ambiguity.md

```diff
@@ -0,0 +1,228 @@
+# Handle Ambiguity Between Protocol Method and Convenience Method
```

Comment:
A title would be something like "Handling ambiguity between protocol and convenience methods". And while this file name doesn't have a space, underscores don't encode well either. Can we just call this "handling-ambiguity.md"?